repos/spack_repo/builtin/packages/mpb/package.py

```diff
@@ -19,6 +19,7 @@ class Mpb(AutotoolsPackage):
     version("1.11.1", sha256="7311fc525214c1184cad3e0626b8540c0b53b3c31c28e61ce6ec2860088eca46")
 
     depends_on("c", type="build")  # generated
+    depends_on("fortran", type="build")  # generated
```

Comment:
`mpb` depends on fortran when it detects a BLAS implementation. What we should likely do here is to create a `+blas` variant (default=False) and update the fortran dependency to be conditional on that variant.

```
variant("blas", default=False, description="Enable BLAS support")

...
...
...

depends_on("fortran", type="build", when="+blas")

...
...

depends_on("blas", when="+blas")
```